### DISCUSSION TITLE
sudo: apt-get: command not found

### DISCUSSION THREAD

**Discussion by adesilvestri** [indomus/forum#654] [Hardware & Firmware]
Salve a tutti,
ho installato Home Assistant è devo configurare il framework di una chiavetta ZIGBEE CC2531.

Seguendo la guida mi dice che devo  installare wiringpi.

Lancio il comando da terminal:
"sudo apt-get install wiringpi"
ma mi viene reso 
"sudo: apt-get: command not found"

Mi potete dare una mano?

Grazie

**Comment by snakuzzo** [comment]
Probabilmente sei su HassOS, non su RaspOS.
Quindi questo che stai provando a fare, non puoi farlo

**Reply by adesilvestri** [reply]
ok, e come posso passare su raspOS?

**Reply by snakuzzo** [reply]
riflashi la SD/SSD e metti RaspOS al posto di HassOS :)

**Reply by KC-inDomus** [reply]
> ok, e come posso passare su raspOS?

puoi anche usare una nuova microSD temporanea, sulla quale installare Raspberry Pi OS, fare il boot, fare ciò che devi fare, per poi rimettere quella originale con HassOS (che, per inciso, prima molli e meglio è).
